src/Mado/QueryBundle/Queries/QueryBuilderFactory.php

```diff
@@ -326,6 +336,7 @@ private function applyFilterOr($filter, $value, $orCondition)
     {
         $whereCondition = null;
         $filterAndOperator = explode('|',$filter);
+        $op = Objects\Operator::fromString($filterAndOperator[1]);
```

Comment:
can we use only Operator::fromString($filterAndOperator[1]); without all path?

Reply:
👍